Backpack-SwiftUI/Rating/README.md

```diff
@@ -0,0 +1,65 @@
+# Rating
+
+[![Cocoapods](https://img.shields.io/cocoapods/v/Backpack-SwiftUI.svg?style=flat)](hhttps://cocoapods.org/pods/Backpack-SwiftUI)
+[![class reference](https://img.shields.io/badge/Class%20reference-iOS-blue)](https://backpack.github.io/ios/versions/latest/swiftui/Structs/BPKRating.html)
+[![view on Github](https://img.shields.io/badge/Source%20code-GitHub-lightgrey)](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack-SwiftUI/Rating)
+
+## Using Title String
+
+| Day | Night |
+| --- | --- |
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone%208-swiftui_rating___with-title-text_lm.png" alt="" width="375" /> |<img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone%208-swiftui_rating___with-title-text_dm.png" alt="" width="375" /> |
+
+## Using Title View
+
+| Day | Night |
+| --- | --- |
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone%208-swiftui_rating___with-custom-title-view-image_lm.png" alt="" width="375" /> |<img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone%208-swiftui_rating___with-custom-title-view-image_dm.png" alt="" width="375" /> |
+
+## Usage
+
+### Rating with a title string
+
+```swift
+BPKRating(
+    accessibilityLabel: "Rated Excellent, 4.5 out of 5. Base on 1,532 reviews",
```

Comment:
Can we update the doc to match the components? 
Please add the .accessibilityLabel("") modifier in this example as well. 

Would be nice if we can also mention it's a must to have this modifier as well. (Below the usage title maybe?)

Reply:
updated in https://github.com/Skyscanner/backpack-ios/commit/b5728fcab3b6490b601458abc8854b3558705543